pkg/services/newrelic.go

```diff
@@ -8,15 +8,21 @@ import (
 	"net/http"
 	"strings"
 	texttemplate "text/template"
+	"time"
 
 	log "github.com/sirupsen/logrus"
 
 	httputil "github.com/argoproj/notifications-engine/pkg/util/http"
 )
 
 type NewrelicOptions struct {
-	ApiKey string `json:"apiKey"`
-	ApiURL string `json:"apiURL"`
+	ApiKey              string        `json:"apiKey"`
+	ApiURL              string        `json:"apiURL"`
+	InsecureSkipVerify  bool          `json:"insecureSkipVerify"`
+	MaxIdleConns        int           `json:"maxIdleConns"`
+	MaxIdleConnsPerHost int           `json:"maxIdleConnsPerHost"`
+	MaxConnsPerHost     int           `json:"maxConnsPerHost"`
+	IdleConnTimeout     time.Duration `json:"idleConnTimeout"`
```

Comment:
We can't use time.Duration for parsing time values - we will need to use a string and then parse the actual `time.Duration` by doing something like this 

```golang
	idleConnTimeout, err := time.ParseDuration(s.opts.IdleConnTimeout)
	if err != nil {
		return errors.Wrap(err, "failed to parse idle connection timeout")
	}
```

and then passing in `idleConnTimeout` to the `NewTransport` function.